### DISCUSSION TITLE
Control button and text colors within home page banner?

### DISCUSSION THREAD

**Discussion by aimeediehl** [knowledgefutures/pubpub#2957] [Support]
Is there a way to control the colors of the primary and secondary buttons, and their text, within the home page banner? I've created a button but it's lost in the image we're using as our banner background, which happens to be nearly the same shade as the button. Some details:

- Our banner text color is set to light but I'm getting black text for a primary button and white text for a secondary button.
- The primary button looks like a lighter shade of our community's dark accent color (moss green) and the secondary button is invisible (our light accent color is white, but the button shows up as white text without any button behind).
<img width="1214" alt="Screenshot 2023-12-18 at 10 26 53 AM" src="https://github.com/pubpub/pubpub/assets/104081219/078512ab-3afa-43e9-8205-6151395e1ff8">

We don't want to change our site header colors (currently set to dark accent for the banner and light accent for the text) or our banner background image.

Any advice or ideas appreciated!

**Comment by kyledenlinger** [comment]
Hi! This can be taken care of with just a bit of custom CSS.

Go to your Community Settings > Details > Customize CSS, then paste in this code:

`.header-component .hero .hero-content .hero-copy .hero-button .bp3-button {
    background-color: #ffffff;
    color: #000000;
}`

You can change the colors to whatever you want. The colors provided will be a simple white button with black outline and text. This should override the default CSS and apply it equally to both Primary and Secondary buttons in the main header.

**Comment by aimeediehl** [comment]
Brilliant! This worked perfectly and the button looks so much better. Thanks a bunch for your help.

**Comment by meeloun** [comment]
When choosing coursework ghostwriting       [http://australiaway.org/a/courseworkdaixie/](http://australiaway.org/a/courseworkdaixie/)       services, it is necessary to pay attention to multiple aspects to ensure that your choice is correct. By carefully identifying and communicating with institutions, we can help you find personalized ghostwriting services that truly meet your needs.
